src/emoji/emojiMap.js

```diff
@@ -2,9 +2,14 @@
tton": "25b6", "billiards": "1f3b1", "backhand_index_pointing_up": "1f446", "hourglass": "231b", "backhand_index_pointing_right": "1f449", "eight": "0038", "japanese_post_office": "1f3e3", "handbag": "1f45c", "arrows_clockwise": "1f503", "pouting_cat_face": "1f63e", "grimacing_face": "1f62c", "new_moon_with_face": "1f31a", "heavy_exclamation_mark": "2757", "squared_new": "1f195", "horse_racing": "1f3c7", "chicken": "1f414", "umbrella_on_ground": "26f1", "arrow_lower_right": "2198", "scorpion": "1f982", "waning_crescent_moon": "1f318", "virgo": "264d", "libra": "264e", "disappointed_face": "1f61e", "sagittarius": "2650", "confounded_face": "1f616", "horizontal_traffic_light": "1f6a5", "bear": "1f43b", "no_mobile_phones": "1f4f5", "women_partying": "1f46f", "calling": "1f4f2", "oncoming_taxi": "1f696", "ogre": "1f479", "computer": "1f4bb", "arrow_down": "2b07", "ferris_wheel": "1f3a1", "grinning_face_with_smiling_eyes": "1f601", "fast_down_button": "23ec", "cat2": "1f408", "cloud_with_lightning_and_rain": "26c8", "icecream": "1f366", "globe_showing_asia_australia": "1f30f", "cross_mark": "274c", "pistol": "1f52b", "email": "2709", "cheese_wedge": "1f9c0", "tent": "26fa", "joy": "1f602", "file_cabinet": "1f5c4", "key": "1f511", "couple_with_heart": "1f491", "low_brightness": "1f505", "santa_claus": "1f385", "ok_woman": "1f646", "space_invader": "1f47e", "cn": "1f1e8", "cl": "1f191", "globe_showing_europe_africa": "1f30d", "cd": "1f4bf", "roasted_sweet_potato": "1f360", "swimmer": "1f3ca", "wavy_dash": "3030", "high_speed_train": "1f684", "cop": "1f46e", "two_hearts": "1f495", "smoking": "1f6ac", "open_file_folder": "1f4c2", "fearful": "1f628", "grinning": "1f600", "left_arrow": "2b05", "bathtub": "1f6c1", "ping_pong": "1f3d3", "timer_clock": "23f2", "lady_beetle": "1f41e", "u5272": "1f239", "rooster": "1f413", "vs": "1f19a", "vulcan_salute": "1f596", "frame_with_picture": "1f5bc", "bullettrain_front": "1f685", "white_circle": "26aa", "squared_ng": "1f196", "balloon": "1f388", "leaf_fluttering_in_wind": "1f343", "speaker": "1f508", "person_in_bed": "1f6cc", "u6307": "1f22f", "arrow_double_down": "23ec", "whale": "1f40b", "hushed_face": "1f62f", "unamused_face": "1f612", "growing_heart": "1f497", "badminton": "1f3f8", "muscle": "1f4aa", "spiral_notepad": "1f5d2", "rocket": "1f680", "camel": "1f42a", "boot": "1f462", "flag_in_hole": "26f3", "flashlight": "1f526", "high_heeled_shoe": "1f460", "desktop_computer": "1f5a5", "ski": "1f3bf", "musical_keyboard": "1f3b9", "boom": "1f4a5", "potable_water": "1f6b0", "sun_behind_cloud": "26c5", "clock1": "1f550", "face_with_thermometer": "1f912", "hugging_face": "1f917", "five": "0035", "clock5": "1f554", "clock6": "1f555", "clock7": "1f556", "clock8": "1f557", "clock9": "1f558", "doughnut": "1f369", "atm_sign": "1f3e7", "linked_paperclips": "1f587", "candy": "1f36c", "two_men_holding_hands": "1f46c", "hotsprings": "2668", "squared_operating_ideograph": "1f23a", "fax_machine": "1f4e0", "bust_in_silhouette": "1f46
```

Comment:
Names are important. Think more about what each name means; I don't think current name is correct.

The textual representations are often called 'emoticons'. So maybe this will be emoticonToEmojiMap?